tests/contrib/tornado/test_tornado_web.py

```diff
@@ -332,7 +332,7 @@ def get_settings(self):
         # distributed_tracing needs to be disabled manually
         return {
             'datadog_trace': {
-                'analytics': True,
+                'analytics_enabled': True,
                 'analytics_sample_rate': 0.5,
```

Comment:
We should have a test that doesn't set `analytics_sample_rate`, since I think it'll actually raise an exception ?